src/Traits/UserTrait.php

```diff
@@ -17,17 +15,13 @@ trait UserTrait
     /**
      * User email.
      *
-     * @Groups({"create"})
```

Comment:
won't the `create` method need these properties to create a user?

Reply:
should send `email` to create a user?

Reply:
Yep, nice catch, fixing.